### DISCUSSION TITLE
View span attributes from the trace view with a single click?

### DISCUSSION THREAD

**Discussion by wjrogers** [datalust/seq-tickets#2191] [Ideas]
In the new trace view, when I click a span, I don't see any way to view that span's attributes without clicking "Find" (which changes the search query and the list of events). I would like to be able to browse through the spans from the trace view itself, without having change my events query. Is there a way to do this now? If not, will you please consider it for a future release?

**Comment by KodrAus** [comment]
Thanks for the suggestion @wjrogers :+1: There isn't currently a way to do this without changing your filter, but it is definitely on our radar for a future release.

**Comment by liammclennan** [comment]
@wjrogers one more thing - if it is a particular property, or set of properties, that you are interested in you can [set them as columns](https://docs.datalust.co/docs/signals#showing-properties-as-columns) and they will be shown in the trace view.

**Reply by wjrogers** [reply]
Good tip, thanks! It's definitely useful to see columns in the trace view.
